packages/contractkit/src/wrappers/LockedGold.ts

```diff
@@ -20,255 +19,89 @@ export interface VotingDetails {
   weight: BigNumber
 }
 
-interface Commitment {
+interface PendingWithdrawal {
   time: BigNumber
   value: BigNumber
 }
 
-export interface Commitments {
-  locked: Commitment[]
-  notified: Commitment[]
-  total: {
-    gold: BigNumber
-    weight: BigNumber
-  }
-}
-
-export enum Roles {
-  Validating = '0',
-  Voting = '1',
-  Rewards = '2',
-}
-
 export interface LockedGoldConfig {
-  maxNoticePeriod: BigNumber
+  unlockingPeriod: BigNumber
 }
 
 /**
  * Contract for handling deposits needed for voting.
  */
 export class LockedGoldWrapper extends BaseWrapper<LockedGold> {
-  /**
-   * Notifies a Locked Gold commitment, allowing funds to be withdrawn after the notice
-   *   period.
-   * @param value The amount of the commitment to eventually withdraw.
-   * @param noticePeriod The notice period of the Locked Gold commitment.
-   * @return CeloTransactionObject
-   */
-  notifyCommitment: (
-    value: string | number,
-    noticePeriod: string | number
-  ) => CeloTransactionObject<string> = proxySend(this.kit, this.contract.methods.notifyCommitment)
-
-  /**
-   * Creates an account.
-   * @return CeloTransactionObject
-   */
-  createAccount: () => CeloTransactionObject<boolean> = proxySend(
-    this.kit,
-    this.contract.methods.createAccount
+  unlock = proxySend(this.kit, this.contract.methods.unlock)
```

Comment:
Like this
```ts
unlock: (value: NumberLike) => CeloTransactionObject<void> = proxySend(this.kit, this.contract.methods.unlock, tupleParser(parseNumber))
```

Things to note:

 1. Extra parameter in `proxySend` that is able to parse input parameters tuple.
 2. For documentation purposes, be explicit about the signature. If not, instead of `unlock(value)` you'll see `unlock(arg_0)`